sdk/python/foundation-models/system/content-safety/deploy_with_content_safety.ipynb

```diff
@@ -0,0 +1,803 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to create an Safety-Enabled LLaMa Online Endpoint\n",
+    "## This notebook will walk you through the steps to create a Safety-Enabled LLaMa Online Endpoint.\n",
+    "### The steps are:\n",
+    "1. Create a Content Safety for moderating the request from user and response from the LLaMa Online Endpoint\n",
```

Comment:
```suggestion
    "1. Create an Azure Content Safety resource for moderating the request from user and response from the LLaMa Online Endpoint\n",
```

Reply:
Let's use **Azure AI Content Safety** everywhere. It's the official product name.

Reply:
Updated.